ui/src/app/components/webcomponents/action-step-form.component.ts

```diff
@@ -50,16 +50,19 @@ import {AddonTestDataFunction} from "../../models/addon-test-data-function.model
 import {AddonTestDataFunctionParameter} from "../../models/addon-test-data-function-parameter.model";
 import {StepActionType} from "../../enums/step-action-type.enum";
 import {ActionTestDataRuntimeVariableSuggestionComponent} from './action-test-data-runtime-variable-suggestion.component';
+import {extractStringByKey} from "../../utils/strings";
 
 @Component({
-  selector: 'app-action-step-form',
+  selector: '' +
+    'app-action-step-form',
   templateUrl: './action-step-form.component.html',
   styles: [],
   host: {
     '(document:click)': 'onDocumentClick($event)',
   },
 })
 export class ActionStepFormComponent extends BaseComponent implements OnInit {
+  readonly extractStringByKey = extractStringByKey;
```

Comment:
why do we need prop?

Reply:
when im trying to access this function in html template , im not able to access that function . by doing like this im able to access the function in html template.

https://stackoverflow.com/questions/40237561/how-to-access-a-constant-in-an-angular-2-component-and-service